app/interactors/get_users_address.rb

```diff
@@ -0,0 +1,7 @@
+class GetUsersAddress
```

Comment:
it can be just 'UserAddress' without `Get`

Reply:
According to this https://mkdev.me/posts/paru-slov-pro-interaktory-v-rails interactor must be named by action